### ISSUE TITLE
Day keywords not returning the next occurance of the specified day

### ISSUE THREAD

**Issue by benjaminheng** [cs2103aug2015-f11-2j/main#42]
Currently day keywords are returning the date of the specified day in the current week. It should return the next occurance of the day, in the next week if necessary.

e.g. if today is tuesday, 'monday' should be next week's monday.

**Comment by benjaminheng** [comment]
fix:

```
diff = dow - currentDow
if diff < 0:
    diff += 7
return currDate.setdays(getdays+diff)
```

**Comment by benjaminheng** [comment]
Closed with 10d2c5ca753cbb405e747a66a1a996986d654ad0.
